src/NovelRenderingService.cpp

```diff
@@ -18,6 +18,24 @@
 #include <fstream>
 #include <sstream>
 
+//DO NOT DELETE THIS, MOVE THIS TO DEBUG SERVICE WHEN IT EXISTS
```

Comment:
Maybe move this into `DebugService.h` since the file already exists?

Reply:
I will move the comment but not implement it as I was unaware this was a 4.3 API. Will resolve that in another branch however.